### DISCUSSION TITLE
Streampipes Backend API via Postman - authorization issue

### DISCUSSION THREAD

**Discussion by mihau1211** [apache/streampipes#2424] [Q&A] [answered]
Hi, I'm facing an issue accessing Streampipes backend API via Postman. 

I tried to access the /streampipes-backend/api/v4/datalake/measurements endpoint but whenever I used the API Token generated by Streampipes -> Profile -> API, the response was 401.

![image](https://github.com/apache/streampipes/assets/73185784/d41e37db-2a63-4821-a192-ed88688005cd)

I'm using the GET method and Bearer token as an authorization method. This is what my request headers look like:

![image](https://github.com/apache/streampipes/assets/73185784/e4920e36-6408-4216-8f15-de78a301a521)

Here is the response:

![image](https://github.com/apache/streampipes/assets/73185784/4f519b6d-6286-4c7b-a126-7ff0245b6252)

Also, the reason that I want to have access to endpoints is that I want to create a custom dashboard outside the streampipes using data processed by streampipes and stored in data lakes. Is it even possible to achieve?

**Comment by dominikriemer** [comment, marked as answer]
Hi @mihau1211 when using the profile token, you can provide the token with the HTTP headers `X-API-USER` (your login email) and `X-API-KEY` (the token generated in the profile view).

Please let me know if this doesn't work!
Dominik

**Reply by wyyolo** [reply]
I have been testing the go client recently and have also found this issue, but I added a request header and still encountered a 401 error.I used the API testing module provided in streampipes ui, but it also showed a 401 error. which may be that I did not use the module correctly.
![postman](https://github.com/apache/streampipes/assets/105999518/93c773c9-9178-4063-8926-693d3522c930)
![ui](https://github.com/apache/streampipes/assets/105999518/ad130f8d-b536-4697-b240-b3622f74a099)

**Reply by mihau1211** [reply]
Try to use email assigned to user instead of username and it should works
